src/components/Pages/ManageWiki/Features/FederatedProperties.vue

```diff
@@ -58,18 +63,22 @@ export default {
       }
     }
   },
+  computed: {
+    isEnabled () {
+      return this.$store.state.wikis.currentWikiSettings.wikibaseFedPropsEnable
+    }
+  },
   methods: {
     enableFeature () {
       const wiki = this.wikiId
       const setting = 'wikibaseFedPropsEnable'
-      const value = 1
+      const value = !this.isEnabled
 
       this.$store.dispatch('updateSetting', { wiki, setting, value })
-        .then(() => this.hideModal())
+        .then(() => this.$store.dispatch('setFederatedPropertiesEnabled', value))
```

Comment:
The store has this information so we could as well just dispatch a "toggleFederatedPropertiesEnabled" action without any value. This is fine too, though.